portfolio/src/main/webapp/me.html

```diff
@@ -1,26 +1,29 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
-    <header>
-      <h1>My Portfolio</h1>
-    <header>
     <div class="topnav">
       <a class="active" href="index.html">Home</a>
       <a href="me.html">About Me</a>
       <a href="travels.html">Travels</a>
+      <a href="yale.html">Yale</a>
     </div>
-    <div>
-      <!-- TODO: About Me page's content -->
+    <header>
       <h1>About Me</h1>
+    <header>
+    <div class="content-body">
+      <!-- TODO(jdstriker731): Add content for 'About Me' page. -->
     </div>
     <footer>
-      <h3>Contact Information</h3>
+      <h3>Contact Information: (607) 752-4042    john.dallard@yale.edu    https://www.linkedin.com/in/jonathan-dallard-5b956b196/</h3>
```

Comment:
hmmm these white spaces will be shortened in the DOM. You should use span tags with classes to space them properly.

Reply:
I noticed when I ran it recently lol. I'll do that instead